resources/cloudformation-stack.go

```diff
@@ -21,24 +22,34 @@ func ListCloudFormationStacks(sess *session.Session) ([]Resource, error) {
 	for _, stack := range resp.Stacks {
 		resources = append(resources, &CloudFormationStack{
 			svc:  svc,
-			name: stack.StackName,
+			stack: stack,
 		})
 	}
 	return resources, nil
 }
 
 type CloudFormationStack struct {
 	svc  *cloudformation.CloudFormation
-	name *string
+	stack *cloudformation.Stack
 }
 
 func (cfs *CloudFormationStack) Remove() error {
 	_, err := cfs.svc.DeleteStack(&cloudformation.DeleteStackInput{
-		StackName: cfs.name,
+		StackName: cfs.stack.StackName,
 	})
 	return err
 }
 
+func (cfs *CloudFormationStack) Properties() types.Properties {
+	properties := types.NewProperties()
+	properties.
+		Set("Name", cfs.stack.StackName)
+	for _, tagValue := range cfs.stack.Tags {
+		properties.Set("tag:"+*tagValue.Key, tagValue.Value);
```

Comment:
key is string... can't be nil? I can check for empty....

Reply:
The [docs](https://godoc.org/github.com/aws/aws-sdk-go/service/cloudformation#Tag) say it is `*string` and not `string`. If it is `nil` the dereference `*tagValue.Key` would cause a panic.

Reply:
got it! :)